pkg/cmd/run/view/view.go

```diff
@@ -230,6 +230,12 @@ func runView(opts *ViewOptions) error {
 		}
 	}
 
+	if err := opts.IO.StartPager(); err == nil {
```

Comment:
How does a pager interpret/interact with the progress indicator? I think the pager was started in this command explicitly after the progress indicator, but I don't actually know if they conflict.